internal/store/postgres/utils.go

```diff
@@ -42,6 +42,9 @@ func translateErr(err error) error {
 		switch pgErr.Code.Name() {
 		case "unique_violation":
```

Comment:
any reason why not comparing it with [pg error code](https://github.com/jackc/pgerrcode/blob/master/errcode.go) so we don't compare it explicitly with string?

Reply:
i have never used that library so it never crossed my mind. also, i feel adding that library is costlier than the benefit it provides (because ultimately it's a string constant somewhere right? it's defined by PG and likely to stay for a long time). 

we could take the error codes that we care about from the lib and add it in our codebase (with link to pg reference in comments) so that it doesn't go through some accidental change.

Reply:
Sorry my bad, I just realized the pgErr is from `lib/pq`. I thought the error code is using `github.com/jackc/pgconn` which I used to use. The `lib/pq` one is essentially has the same error code mapping so I guess there is no problem with this.